doc/classes/ReferenceRect.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ReferenceRect" inherits="Control" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Reference frame for GUI.
+		A rectangle hint for designing UIs.
 	</brief_description>
 	<description>
-		A rectangle box that displays only a [member border_color] border color around its rectangle. [ReferenceRect] has no fill [Color]. If you need to display a rectangle filled with a solid color, consider using [ColorRect] instead.
+		A rectangle box that displays only a colored border around its rectangle. It is used to visualize the extents of a [Control].
```

Comment:
```suggestion
		A rectangle box that displays only a colored border around its area. It is used to visualize the extents of a [Control], but it can also be used as a colored frame display.
```
Note the `editor_only` property.

Reply:
From an earlier comment:

> The ability to show up in-game wasn't even present in older versions, and editor-only is by default. Even the "in-game display" I would say is also meant for debugging purposes and not to have borders in the final product.

Suppose if Marker2/3D had an option to show its gizmos in the running project (speaking of which, this would be pretty darn useful) - I still wouldn't put in the documentation that it can be used to draw crosses.

I'll leave this unresolved for now.